tests/karate/src/test/java/common/net/MultiMsgWebSocketClient.java

```diff
@@ -37,4 +41,27 @@ public synchronized Object listen(long timeout) {
   public MessageBuffer getBuffer() {
     return queue;
   }
+
+  public void publish(String data, int id, String channel){
+    JsonConverter jsonConverter = new JsonConverter();
+    Json request =  jsonConverter.publishМessageFromData(data, id, channel);
+    this.send(request.toString());
+  }
+
+  public String getBackendResponseWithBroadcast(String a){
+    String broadcast = getBuffer().takeTimeout(5000);
```

Comment:
what do you do with the broadcast? how do you assert that you had the right result ?